src/integrationTest/java/uk/gov/hmcts/darts/common/util/Utils.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.darts.common.util;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+public final class Utils {
```

Comment:
could use Lomboks @Utility annotation, then don't need private constructor etc.

Reply:
I tried to use Lombok but PMD is complaining at annotation usage.